### PR TITLE
Sqlite for RP, Views, and Controllers

### PR DESCRIPTION
**Pull request by deepchoudhery**
Cherry-picking over from release/3.1 branch.

### CODE REVIEW

**Comment by deepchoudhery** [comment]
@Rick-Anderson These docs will need updating for sqlite flag for controllers and razorpages.
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/tools/dotnet-aspnet-codegenerator?view=aspnetcore-3.1

File: src/VS.Web.CG.EFCore/MessageStrings.resx [hunk 1]
```diff
@@ -144,6 +144,9 @@
   <data name="DbContextTypeNotFound" xml:space="preserve">
     <value>Could not get the reflection type for DbContext : {0}</value>
   </data>
+  <data name="EFSqlitePackageNotAvailable" xml:space="preserve">
```
Comment (deepchoudhery) [inline src/VS.Web.CG.EFCore/MessageStrings.resx]: will update these

File: src/VS.Web.CG.EFCore/MessageStrings.resx [hunk 2]
```diff
@@ -145,10 +145,10 @@
     <value>Could not get the reflection type for DbContext : {0}</value>
   </data>
   <data name="EFSqlitePackageNotAvailable" xml:space="preserve">
-    <value>Please install the package 'Microsoft.EntityFrameworkCore.Sqlite' and retry again.</value>
+    <value>Install the package 'Microsoft.EntityFrameworkCore.Sqlite' and retry again.</value>
```
Comment (Rick-Anderson) [inline src/VS.Web.CG.EFCore/MessageStrings.resx]:
```suggestion
    <value>Install the package 'Microsoft.EntityFrameworkCore.Sqlite' and retry.</value>
```

File: src/VS.Web.CG.EFCore/MessageStrings.resx [hunk 3]
```diff
@@ -145,10 +145,10 @@
...
   </data>
   <data name="EFSqlServerPackageNotAvailable" xml:space="preserve">
-    <value>Please install the package 'Microsoft.EntityFrameworkCore.SqlServer' and retry again.</value>
+    <value>Install the package 'Microsoft.EntityFrameworkCore.SqlServer' and retry again.</value>
```
Comment (Rick-Anderson) [inline src/VS.Web.CG.EFCore/MessageStrings.resx]:
```suggestion
    <value>Install the package 'Microsoft.EntityFrameworkCore.SqlServer' and retry.</value>
```

File: src/VS.Web.CG.Design/Resources.resx [hunk 4]
```diff
@@ -142,7 +142,7 @@
     <value>Path to .csproj file in the project.</value>
   </data>
   <data name="RunWithoutSimulationMode" xml:space="preserve">
-    <value>Please run scaffolding without --simulation-mode switch</value>
+    <value>Run scaffolding without --simulation-mode switch</value>
```
Comment (Rick-Anderson) [inline src/VS.Web.CG.Design/Resources.resx]:
```suggestion
    <value>Run scaffolding without the --simulation-mode switch.</value>
```

File: src/VS.Web.CG.Mvc/MessageStrings.resx [hunk 5]
```diff
@@ -154,19 +154,19 @@
...
-    <value>To scaffold controllers and views using models please install Entity Framework core packages and try again: {0}</value>
+    <value>To scaffold controllers and views using models, install Entity Framework core packages and try again: {0}</value>
   </data>
   <data name="InstallPackagesForScaffoldingIdentity" xml:space="preserve">
-    <value>Please install the following packages to your project for scaffolding identity: {0}</value>
+    <value>Install the following packages to your project for scaffolding identity: {0}</value>
   </data>
   <data name="InstallPackageMessage" xml:space="preserve">
-    <value>Please install the below packages to your project:</value>
+    <value>Install the below packages to your project:</value>
```
Comment (Rick-Anderson) [inline src/VS.Web.CG.Mvc/MessageStrings.resx]:
```suggestion
    <value>Install the following packages to your project:</value>
```

**Pull request merged**